### PR TITLE
Added functionality to check for newer release

### PR DESCRIPTION
**Pull request by vasily-pozdnyakov**
### Summary of changes

There is a new top bar menu item to check for OpossumUI latest release.

### Context and reason for change

Part of #1472

### How can the changes be tested

Unit tests.
Check in the UI: "help" - "check for updates"

### CODE REVIEW

**Review by leslielazzarino — APPROVED** [review]
Cool. It would be even cooler if it checked for a new version when the app is started.

File: src/Frontend/Components/GlobalPopup/__tests__/GlobalPopup.test.tsx [hunk 1]
```diff
@@ -210,4 +219,18 @@ describe('The GlobalPopUp', () => {
     const header = 'Warning: Outdated input file format';
     expect(screen.getByText(header)).toBeInTheDocument();
   });
+
+  it('opens the UpdateAppPopup', () => {
```
Comment (benedikt-richter) [inline src/Frontend/Components/GlobalPopup/__tests__/GlobalPopup.test.tsx:223]: We should also test the query.

File: src/Frontend/Components/UpdateAppPopup/__tests__/UpdateAppPopup.test.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import React from 'react';
+import { renderComponentWithStore } from '../../../test-helpers/render-component-with-store';
+import { UpdateAppPopup } from '../UpdateAppPopup';
+import { screen } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import commitInfo from '../../../../commitInfo.json';
+
+describe('UpdateAppPopup', () => {
```
Comment (benedikt-richter) [inline src/Frontend/Components/UpdateAppPopup/__tests__/UpdateAppPopup.test.tsx:16]: Nice tests! 👍

**Review by benedikt-richter — CHANGES_REQUESTED** [review]
Screenshot 2023-03-31 at 10 30 39
Screenshot 2023-03-31 at 10 33 06

works nicely!

Some UI observations: (see screenshots)
* The spinner is too big in my opinion.
* The font is different in the link

File: src/Frontend/Components/UpdateAppPopup/__tests__/UpdateAppPopup.test.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
+// SPDX-FileCopyrightText: Meta Platforms, Inc. and its affiliates
+// SPDX-FileCopyrightText: TNG Technology Consulting GmbH <https://www.tngtech.com>
+// SPDX-FileCopyrightText: Nico Carl <nicocarl@protonmail.com>
```
[the remark this answers is not in the document]
Comment (vasily-pozdnyakov) [reply src/Frontend/Components/UpdateAppPopup/__tests__/UpdateAppPopup.test.tsx:3]: Copied from some other component

**Comment by vasily-pozdnyakov** [comment]
> Screenshot 2023-03-31 at 10 30 39 Screenshot 2023-03-31 at 10 33 06
> 
> works nicely!
> 
> Some UI observations: (see screenshots)
> 
> * The spinner is too big in my opinion.
> * The font is different in the link

I am ok with the spinner size - anyway it is usually shown just for a second.
I corrected the font.

**Review by benedikt-richter — APPROVED** [review]

**Pull request merged**